modules/twinklespeedy.js

```diff
@@ -1516,6 +1515,15 @@ Twinkle.speedy.callbacks = {
 				}
 			}
 
+			// Scribunto isn't parsed like wikitext, so CSD templates on modules need special handling to work
+			if (params.scribunto) {
+				var equals = '';
+				while(code.indexOf(']' + equals + ']') !== -1) {
+					equals += '=';
+				}
+				code = "require('Module:Module wikitext')._addText([" + equals + '[' + code + ']' + equals + ']);';
```

Comment:
What are the brackets for?  This ends up with something like `require('Module:Module wikitext')._addText([[{{db-vandalism|help=off}}]]);`, which isn't what your doc suggests.

Reply:
Lua uses brackets for the same purpose that most languages use triple-quoted strings for. By putting everything inside of `[[`, `[=[`, etc., it forms a string literal where nothing needs to be escaped. I thought this was simpler/easier than using regular strings and then having to escape things, but let me know if you'd rather I do that instead.

Reply:
No, that's excellent; there's user-entered text (esp. with `{{db}}`) so that's fabulous.

Presumably it's just different in the on-wiki doc to just KISS when for manual editing?

Reply:
Yep, that's exactly why. I expect that most editors following that documentation will (1) be more familiar with regular strings with escaping, and (2) will probably almost always use things that don't need any escaping.